client/src/app/core/services/projector.service.ts

```diff
@@ -103,4 +205,60 @@ export class ProjectorService extends OpenSlidesComponent {
             return projector.projectiondefaults.map(pd => pd.name).includes(projectiondefault);
         });
     }
+
+    public getModelFromProjectorElement<T extends BaseModel>(element: IdentifiableProjectorElement): T {
+        if (!this.slideManager.canSlideBeMappedToModel(element.name)) {
+            throw new Error('THis projectorelement cannot be mapped to a model');
+        }
+        const identifiers = element.getIdentifiers();
+        if (!identifiers.includes('name') || !identifiers.includes('name')) {
+            throw new Error('To map this element to a model, a name and id is needed.');
+        }
+        return this.DS.get<T>(element.name, element.id);
+    }
+
+    public async projectNextSlide(projector: Projector): Promise<void> {
```

Comment:
many methods have the prefix "project".
Would it be misleading to just name the methos `nextSlide` and `previewSlide`, `previousSlide`